modules/ROOT/pages/Development/ModLoader/Subsystems.adoc

```diff
@@ -19,39 +19,20 @@ For example, the mod Light It Up uses subsystems to create a lamp group system.
 With this system, every lamp can get information about the current group settings
 without storing that data again in each lamp instance.
 
-== Using UModSubsystemHolder
 
-If you want to have a subsystem you will need at least one class
-that inherits the `UModSubsystemHolder` class.
-These holders will spawn and store the references to the subsystems your mod uses.
+== Spawning a Subsystem
 
-It has an `InitSubsystems` function in which you need to spawn the
-subsystems you want to add and then store the references in attributes.
+A subsystem itself is an actor inheriting from `SML.ModSubsystem`.
+You can also use subsystems to store data to save states.
 
-To register a subsystem holder, you should add it to the `Mod Subsystems` array of the
-xref:Development/ModLoader/ModModules.adoc#_game_instance_module_ugameinstancemodule[Game Instance Module].
+The subsystem should be registered as a property in `SML.GameWorldModule`
 
-You can use the static `K2_GetModSubsystemHolder` function
-to get your holders from other places in the code.
-This function returns a reference to your holder,
-which itself has attributes references to the subsystems you initialized in it.
+image:Satisfactory/Subsystems/Subsystems_gwm.PNG[GameWorldModule]
 
-== Spawning a Subsystem
+You should use the `GetSubsystemActor` function to reference your subsystem.
 
-A subsystem itself is an actor inheriting from `AFGSubsystem`.
-You can also use subsystems to store data to save states.
+image:Satisfactory/Subsystems/Subsystems_sam.PNG[GameWorldModule]
+
+* The first parameter of the function is a reference to your `SubsystemActorManager` acquired using the function `GetSubsystemActorManager`.
```

Comment:
It would be best to say that the subsystems are held by the SubsystemActorManager, show how to get that, and then how to get the desired subsystem out